### ISSUE TITLE
The dependency library is up-to-date and cannot be dragged on Android

### ISSUE THREAD

**Issue by lh9403** [computerjazz/react-native-draggable-flatlist#377]
Use the latest:  react-native-reanimated, react-native-gesture-handler, can't drag on Android. Did anyone encounter it

Platform: Android
react-native-draggable-flatlist version: 3.1.1
"react": "17.0.2",
"react-native": "0.67.4",
"react-native-reanimated": "2.8.0",
"react-native-gesture-handler": "2.4.1",

**Comment by NensiKasundra** [comment]
Yes, it's not working with the new version in android only in iOS it's working fine. Please check for the below version.


  "react": "^18.0.0",
  "react-native": "0.68.1",
  "react-native-gesture-handler": "^2.4.0",
  "react-native-reanimated": "^2.7.0",
  "react-native-draggable-flatlist": "^3.1.1",

**Comment by NensiKasundra** [comment]
Sloved. Here is the gesture handler issue. so I wrapped View with the below code

import { GestureHandlerRootView } from 'react-native-gesture-handler';

<GestureHandlerRootView style={{ flex: 1 }}>
      //your draggableList Code 
</GestureHandlerRootView>

**Comment by lh9403** [comment]
> Solved. Here is the gesture handler issue. so I wrapped View with the below code
> 
> ```
> import { GestureHandlerRootView } from 'react-native-gesture-handler';
> 
> <GestureHandlerRootView style={{ flex: 1 }}>
>       //your draggableList Code 
> </GestureHandlerRootView>
> ```
> 
> You can also do the same in your APP.js also so you don't have to implement it with a separate component.

Thank you. This modification works 😄
